lib/ansible/modules/dnf.py

```diff
@@ -443,7 +443,8 @@ def _sanitize_dnf_error_msg_remove(self, spec, error):
         """
         if (
             'no package matched' in to_native(error) or
-            'No match for argument:' in to_native(error)
+            'No match for argument:' in to_native(error) or
+            'All matches were filtered out by exclude filtering for argument:' in to_native(error)
```

Comment:
We should try and remove scraping text on exceptions rather than adding more cases. My attempt https://github.com/ansible/ansible/pull/83297.